packages/react-admin-core/src/table/excelexport/useExportTableQuery.tsx

```diff
@@ -0,0 +1,45 @@
+import { useApolloClient } from "@apollo/react-hooks";
+import * as React from "react";
+import { Table } from "../Table";
+import { ITableQueryApi } from "../TableQueryContext";
+import { createExcelExportDownload } from "./createExcelExportDownload";
+import { IExportApi } from "./IExportApi";
+
+export function useExportTableQuery<IVariables>(
+    api: ITableQueryApi,
+    variables: IVariables,
+    fileName?: string,
+    worksheetName?: string,
```

Comment:
da wär ich ja dafür teile (oder alles) in options objekt zu packen

Reply:
das hab ich auf resolveTableData geändert